content/blog/journaling/index.md

```diff
@@ -0,0 +1,323 @@
+---
+title: "Speeding up Pulumi deployments by up to 20x"
+
+date: 2025-12-08T17:57:55+02:00
+
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: Pulumi deployments get up to 20x faster with journaling, a new snapshotting approach that speeds up large stacks while keeping full data integrity.
+
+meta_image: meta.png
+
+authors:
+    - thomas-gummerer
+
+tags:
+    - journaling
+    - performance
+    - data-integrity
+
+social:
+    twitter: |
+        Pulumi deployments just got up to 20x faster:
+
+        - Journaling: Send only changes, not full snapshots
+        - Data integrity: No compromise on reliability
+        - Network traffic cut by 85%+ on large stacks
+
+        Try it: [link]
+    linkedin: |
+        # Pulumi Deployments Get Up to 20x Faster
+
+        Large Pulumi stacks just got a major performance boost. Here's what changed.
+
+        # The Problem
+        Pulumi saves snapshots at every deployment step to maintain data integrity. For large stacks, this creates a bottleneck: uploading the full snapshot serially slows everything down.
+
+        # The Solution: Journaling
+        Instead of sending the whole snapshot, journaling sends only individual changes. These journal entries can go in parallel, and Pulumi Cloud reconstructs the full snapshot on the backend.
```

Comment:
```suggestion
        Instead of sending the whole snapshot, journaling sends only individual changes. These journal entries can be written in parallel, and Pulumi Cloud can reconstruct the full snapshot on the backend.
```